utils/templates/interfaces.py.tpl

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Dict, List, Literal, Mapping, Union
+from elasticsearch_dsl.document_base import InstrumentedField
+from elasticsearch_dsl import function as f, interfaces as i, Query
+from elasticsearch_dsl.utils import AttrDict, NotSet, NOT_SET
+
+PipeSeparatedFlags = str
```

Comment:
For now I'm not worrying about `PipeSeparatedFlags` and consider it a string. But we could come up with a more Pythonic way to generate a list of flags.